frontend/src/App.js

```diff
@@ -12,4 +14,4 @@ const App = () => (
   </BrowserRouter>
 );
 
-export default App;
+export default DragDropContext(HTML5Backend)(App);
```

Comment:
How does it work?

Reply:
It's required by react-big-scheduler library to display the events in the schedule and enable drag and drop features. Usage is explained here: http://react-dnd.github.io/react-dnd/docs/api/drag-drop-context